packages/shared/src/graphql/squads.ts

```diff
@@ -461,6 +470,10 @@ const formToInput = (form: SquadForm): SharedSquadInput => ({
   memberPostingRole: form.memberPostingRole,
   memberInviteRole: form.memberInviteRole,
   categoryId: form.categoryId,
+  moderationRequired:
+    form.memberPostingRole === SourceMemberRole.Moderator
+      ? false
+      : form.moderationRequired,
```

Comment:
According to design we want the switch to still show "on" when it's disabled, so we have to check manually if the posting role is set to "moderators"

Reply:
That doesn't seem right? 😅
Might just be a design mistake no?